proposals/dao/v2Phase1-part1.ts

```diff
@@ -0,0 +1,144 @@
+         moves pcv from the old eth & dpi uni pcv deposits into the new ones.
+
+----- PART 1 -----
+
+DEPLOY ACTIONS:
+1. ETH Uni PCV Deposit
+2. DPI Uni PCV Deposit
+3. ETH Bonding Curve
+4. Ratio PCV Controller
+
+DAO ACTIONS:
+1. Grant Minter role to new ETH BondingCurve
+2. Grant Minter role to new ETH Uni PCV Deposit
+3. Grant Minter role to new DPI Uni PCV Deposit
+4. Grant PCV Controller role to new RatioPCVController
+5. Set ETH Bonding Curve Minting Cap Max
+6. Update DPI Bonding Curve allocation
+7. Move PCV from old ETH Uni PCV Deposit to new
+8. Move PCV from old DPI Uni PCV Deposit to new
+9. Sets allocation on the DPI bonding curve.
+10. Sets mint cap on the new eth bonding curve.
+11. Revokes the minter role from the old uniswap pcv deposit.
+12. Revokes the minter role from the old dpi uniswap pcv dpeosit.
+13. Revokes the minter role from the old eth bonding curve.
+14. Calls deposit() on the dpi uniswap pcv deposit.
+15. Calls deposit() on the old uniswap pcv deposit.
+16. Revokes the pcv controller role from the old ratio pcv controller.
+17. Revokes the pcv controller role from the old uniswap pcv controller.
+18. Revokes the minter role from the old uniswap pcv controller.
+
+*/
+
+export const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  const { timelock } = addresses;
+
+  await hre.network.provider.request({
+    method: 'hardhat_impersonateAccount',
+    params: [timelock]
+  });
+};
+
+export const run: RunUpgradeFunc = async (addresses, oldContracts, contracts, logging = false) => {
+  const rariPool19DpiPCVDeposit = contracts.rariPool19DpiPCVDeposit as ERC20CompoundPCVDeposit;
+  const oldRatioPCVController = contracts.ratioPCVController as RatioPCVController;
+  const dpiUniswapPCVDeposit = contracts.dpiUniswapPCVDeposit as UniswapPCVDeposit;
+  const uniswapPCVDeposit = contracts.uniswapPCVDeposit as UniswapPCVDeposit;
+  const bondingCurve = contracts.bondingCurve as EthBondingCurve;
+  const tribeReserveStabilizer = contracts.tribeReserveStabilizer as TribeReserveStabilizer;
+  const ratioPCVController = contracts.ratioPCVController as RatioPCVController;
+  const pcvEquityMinter = contracts.pcvEquityMinter as PCVEquityMinter;
+  const collateralizationOracleKeeper = contracts.collateralizationOracleKeeper as CollateralizationOracleKeeper;
+  const dpiBondingCurve = contracts.dpiBondingCurve as BondingCurve;
+  const core = contracts.core as Core;
+
+  logging && console.log('Granting Minter role to new BondingCurve');
+  await core.grantMinter(bondingCurve.address);
+
+  logging && console.log('Granting Minter role to new DPI UniswapPCVDeposit');
+  await core.grantMinter(dpiUniswapPCVDeposit.address);
+
+  logging && console.log('Granting Minter role to new UniswapPCVDeposit');
+  await core.grantMinter(uniswapPCVDeposit.address);
+
+  logging && console.log('Granting Burner role to new TribeReserveStabilizer');
+  await core.grantBurner(tribeReserveStabilizer.address);
```

Comment:
Not going to be needed, but can be fixed in another PR

Reply:
Actually this contract shouldn't exist yet

Reply:
Removed.